src/services/auth-header.js

```diff
@@ -0,0 +1,11 @@
+const authHeader = () => {
+    const user = JSON.parse(localStorage.getItem('user'));
+    if (user && user.Token) {
+        // todo: fix this part to take current user token. currently using a hard coded one - return { Authorization: 'Bearer ' + user.Token };
```

Comment:
here like you said, this is really important, just check in local storage and then use your Bearer token.

Reply:
Thanks, fixed.